.github/ISSUE_TEMPLATE/graduation.md

```diff
@@ -0,0 +1,35 @@
+# Application for Graduation
+- [ ] Have committers from at least two organizations.
+- [ ] Have achieved and maintained a Core Infrastructure Initiative Best Practices Badge.
+- [ ] Have completed an independent and third party security audit with results published of similar scope and quality as this example which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before graduation.
+- [ ] Explicitly define a project governance and committer process. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
+- [ ] Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website). For a specification, have a list of adopters for the implementation(s) of the spec.
+
+
+## About your Project
+
+\[INSERT: Project name]
+\
+[INSERT: Link to project website with summary of what the project does] 
+
+[INSERT: Link to GitHub repo] 
+
+[INSERT: Link to Incubation doc]
+
+## Why is the Project is ready to Graduate? 
+
+\[INSERT: ]
+
+## What has changed since you joined Incubation? 
+
+\[INSERT: new processes / governance? end user adoption? community size? Feature stability?]
+
+## How can your project survive its founders and founding companies?
+
+\[INSERT: setting yourself up for the long run\]
+
+## Who are the best advocates for your project? What do they say?
+
+\[INSERT: grass roots? adopters?\]
+
+[What do your enemies say too?\]
```

Comment:
Is this line needed? is it trying to ask what other projects or vendors think of the project? If so, I'd consider rewording it...enemies is a strong word.